plutus-conformance/test-cases/uplc/evaluation/untyped-plutus-core-tests/iteAtString/iteAtString.uplc

```diff
@@ -0,0 +1 @@
+(program 1.0.0 (force (builtin ifThenElse)))
```

Comment:
e.g. this is obviously a stupid test :D

Reply:
I'm not so sure.  That would fail if (immediately) it was `(force (builtin modInteger))` or something, so it is testing something.  Maybe we need a test for the second case too!